infrastructure/galasa-plan-b-lon02/argocd/argocd.yaml

```diff
@@ -9909,7 +9909,7 @@ spec:
         - ""
         - --appendonly
         - "no"
-        image: redis:7.0.4-alpine
+        image: harbor.galasa.dev/docker_proxy_cache/library/redis:7.0.4-alpine
```

Comment:
I think we can leave this one as it comes with the ArgoCD configuration - can you revert back please?